pkg/dogstatsd/replay/writer.go

```diff
@@ -70,10 +71,9 @@ type TrafficCaptureWriter struct {
 }
 
 // NewTrafficCaptureWriter creates a TrafficCaptureWriter instance.
-func NewTrafficCaptureWriter(l string, depth int) *TrafficCaptureWriter {
+func NewTrafficCaptureWriter(depth int) *TrafficCaptureWriter {
 
 	return &TrafficCaptureWriter{
-		Location:    l,
 		Traffic:     make(chan *CaptureBuffer, depth),
```

Comment:
`Location` field can be removed from `TrafficCaptureWriter`.

Reply:
It's actually still used, but it's set when `Capture()` is called. I'll keep it. Thanks for keeping me honest. ;)

Reply:
Isn't `Location ` be used only in the function `Capture()`?

Reply:
Yes, but I think I want to keep it around regardless. If you have a strong opinion I can remove it.